frontend/cypress/integration/examples/core/dataframe_dimension_spec.spec.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// <reference types="cypress" />
+
+describe('Dataframes with different sizes', () => {
+  const expected = [
+    { width: '727px', height: '300px' },
+    { width: '247px', height: '150px' },
+    { width: '247px', height: '300px' },
+    { width: '727px', height: '150px' },
+
+  ]
+  before(() => {
+    // http://gs.statcounter.com/screen-resolution-stats/desktop/worldwide
+    cy.viewport(1366, 768)
```

Comment:
Don't need this line since we set the dimensions for all tests in [cypress.json](https://github.com/streamlit/streamlit/blob/develop/frontend/cypress.json#L9)